control-plane/p4RuntimeSerializer.cpp

```diff
@@ -1156,29 +1156,34 @@ static void collectTableSymbols(P4RuntimeSymbolTable& symbols,
 }
 
 static void collectParserSymbols(P4RuntimeSymbolTable& symbols,
+                                 P4RuntimeArchHandlerIface* archHandler,
                                  const IR::ParserBlock* parserBlock) {
     CHECK_NULL(parserBlock);
 
     auto parser = parserBlock->container;
     CHECK_NULL(parser);
 
+    auto name = archHandler->getControlPlaneName(parserBlock);
+
     for (auto s : parser->parserLocals) {
         if (auto inst = s->to<IR::P4ValueSet>()) {
-            symbols.add(P4RuntimeSymbolType::VALUE_SET(), inst);
+            // symbols.add(P4RuntimeSymbolType::VALUE_SET(), inst);
+            symbols.add(P4RuntimeSymbolType::VALUE_SET(), name);
```

Comment:
that doesn't look right: you are using the name of the parser here, not the name of the PVS